fluid/voice.cpp

```diff
@@ -193,6 +193,18 @@ float Voice::gen_get(int g)
       return gen[g].val;
       }
 
+void Voice::calcVolEnv(int n,int restN,fluid_env_data_t *env_data)
```

Comment:
Style, should be `void calcVolEnv(int n, int restN, fluid_env_data_t* env_data)`